test/e2e/children-page/index.test.ts

```diff
@@ -0,0 +1,23 @@
+import { createNextDescribe } from 'e2e-utils'
+
+createNextDescribe(
```

Comment:
Nit: should we nest this under `test/e2e/app-dir`/ include it in an existing fixture?

Reply:
Seems this test also passes without the changes here 🤔